src/EVault/IEVault.sol

```diff
@@ -401,14 +416,23 @@ interface IGovernance {
 
     /// @notice Set a new LTV config
     /// @param collateral Address of collateral to set LTV for
-    /// @param ltv New LTV in 1e4 scale
+    /// @param borrowLTV New borrow LTV for originating positions in 1e4 scale
+    /// @param liquidationLTV New liquidation LTV after ramp ends in 1e4 scale
     /// @param rampDuration Ramp duration in seconds
-    function setLTV(address collateral, uint16 ltv, uint32 rampDuration) external;
+    function setLTV(address collateral, uint16 borrowLTV, uint16 liquidationLTV, uint32 rampDuration) external;
 
     /// @notice Completely clears LTV configuratrion, signalling the collateral is not considered safe to liquidate anymore
     /// @param collateral Address of collateral
     function clearLTV(address collateral) external;
 
+    /// @notice Set a new maximum liquidation discount
+    /// @param newDiscount New maximum liquidation discount in 1e4 scale
+    function setMaxLiquidationDiscount(uint16 newDiscount) external;
+
+    /// @notice Retrieves liquidation cool off time, which must elapse after successful account status check before account can be liquidated
+    /// @param newCoolOffTime The new liquidation cool off time in seconds
+    function setLiquidationCoolOffTime(uint16 newCoolOffTime) external;
```

Comment:
Suggestion: consider adding a `@dev` comment to cover the scenario when the `newCoolOffTime` is equal to `0`: users will be able to be liquidated in the same block